### DISCUSSION TITLE
Mitkä korkeakoulut hyödyntävät tai viittaavat Käytännöllistä Latexia -oppaaseen opinnäytetyön ohjeistuksessa?

### DISCUSSION THREAD

**Discussion by markosu** [tlikonen/latex-opas#15] [Keskustelu]
Kiinnostaisi tietää kuinka moni korkeakoulu on tietoinen Latexin mahdollisuuksista ja viittaavat tähän oppaaseen opinnäytetöiden ohjeistuksessa tai muualla.

**Comment by tlikonen** [comment]
Samoin. Minuakin kiinnostaisi. Veikkaisin, ettei mikään korkeakoulu tai muukaan taho viittaa tähän oppaaseen, ainakaan vielä, koska opas on aika uusi ja sitä ei varmaankaan tunneta juuri missään.

**Reply by tlikonen** [reply]
Opas on löydettävissä oppilaitosten Finna-järjestelmästä:
https://finna.fi/Record/aoe.1878?sid=4044683201

**Comment by markosu** [comment]
Kuinkahan paljon jumppaa vaatisi se ,että korkeakoulu tekisi valmiin Latex pohjan opinnäytetöille?
Mitä etuja sillä saavutettaisiin?

Oletan, että tämän oppaan avulla opiskelija pystyisi suoriutumaan pohjan käytöstä, jolloin toista opasta ei välttämättä tarvita. Silloin kaikki voittaisivat. :)
